recipes-core/packagegroups/packagegroup-ni-desirable.bb

```diff
@@ -52,6 +52,7 @@ RDEPENDS_${PN} += "\
 	rsync \
 	sshpass \
 	strace \
+	sysconfig-settings-ui \
```

Comment:
If this is a dependency of NI-owned components, then it needs to go into the [`ADDITIONAL_CORE_FEED_RECIPES`](https://dev.azure.com/ni/DevCentral/_git/ni-central?path=/src/rtos/rtoslegacyd/NIOpenEmbedded/x64_dkms.mak&version=GBmain&line=11&lineEnd=12&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents) variable, rather than the `-desirable` packagegroup.

Reply:
Got it. I'll make the change there. Closing this PR.